### PR TITLE
Improve document layouts

### PR DESCRIPTION
**Pull request by GuruprasadAnk1996**
ui-dev-workshop (Phase 3)

### CODE REVIEW

File: elements/document/file/nuxeo-file-metadata-layout.html [hunk 1]
```diff
@@ -30,6 +30,11 @@
       <div name="title">[[document.properties.dc:title]]</div>
     </div>
 
+    <div role="widget" hidden$="[[!document.properties.dc:source]]">
+      <label>[[i18n('label.dublincore.source')]]</label>
+      <div name="source" class="multiline">[[document.properties.dc:source]]</div>
```
Comment (rakeshkumar1019) [inline elements/document/file/nuxeo-file-metadata-layout.html]: do we need this class class="multiline" ?

**Review by rakeshkumar1019 — COMMENTED** [review]
no need to push this files
- package-lock.json
- packages/nuxeo-web-ui-ftest/package-lock.json
- plugin/a11y/package-lock.json

File: ftest/features/create_doc.feature [hunk 2]
```diff
@@ -24,6 +25,7 @@ Feature: Create Document
     And I can see <doctype> metadata with the following properties:
       | name         | value                            |
       | title        | my title                         |
+      | source       | my source                        |
```
Comment (Nisha-QA) [inline ftest/features/create_doc.feature:28]: Looks good

File: elements/document/file/nuxeo-file-create-layout.html [hunk 3]
```diff
@@ -33,6 +33,14 @@
       required
     >
     </nuxeo-input>
+    <nuxeo-input
```
Comment (Gabez0r) [inline elements/document/file/nuxeo-file-create-layout.html:37]: nit: please leave a space between widgets, for consistency with the rest of the file:
```suggestion

    <nuxeo-input
```

File: elements/document/file/nuxeo-file-edit-layout.html [hunk 4]
```diff
@@ -33,6 +33,14 @@
       required
     >
     </nuxeo-input>
+    <nuxeo-input
```
Comment (Gabez0r) [inline elements/document/file/nuxeo-file-edit-layout.html:37]: Same here.

File: elements/document/file/nuxeo-file-import-layout.html [hunk 5]
```diff
@@ -33,6 +33,14 @@
       required
     >
     </nuxeo-input>
+    <nuxeo-input
```
Comment (Gabez0r) [inline elements/document/file/nuxeo-file-import-layout.html:37]: Same here.

**Review by Gabez0r — COMMENTED** [review]
Looking good. Please make sure the unit and functional tests are passing.

**Comment by Gabez0r** [comment]
Also, please do not push the package-lock.json files, as we already lock them when we produce a release candidate, and this will interfere with the process.

**Pull request closed without merging**